src/migrations/1717750717135-extendMimeTypes.ts

```diff
@@ -0,0 +1,16 @@
+import { DEFAULT_ALLOWED_MIME_TYPES } from '@common/enums/mime.file.type';
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class extendMimeTypes1717750717135 implements MigrationInterface {
```

Comment:
Please add the DEFAULT_ALLOWED_MIME_TYPES to this file. Sounds counter-intuitive, but the migration is snapshot-in-time - if you refer a continuously-updated list as the source of data, you will potentially get different outcomes running this at different times.

Reply:
Done.